src/main/java/com/textmessenger/service/NotificationServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.textmessenger.service;
+
+import com.textmessenger.model.entity.Notification;
+import com.textmessenger.model.entity.User;
+import com.textmessenger.repository.NotificationRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class NotificationServiceImpl implements NotificationService {
+  private final NotificationRepository notificationRepository;
+
+  NotificationServiceImpl(NotificationRepository notificationRepository) {
+    this.notificationRepository = notificationRepository;
+  }
+
+  @Override
+  public Notification createNotification(String type, User user, Long id) {
+    Notification notification = new Notification();
+    notification.setContentId(id);
+    notification.setUser(user);
+    notification.setType(type);
+    return notificationRepository.save(notification);
+
+  }
+
+  @Override
+  public void updateNotification(Long id) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. ([squid:S1186](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1186))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=139&issues=AWV3RcokelKyZq-pEF1Y&open=AWV3RcokelKyZq-pEF1Y)

[ISSUE_KEY]: AWV3RcokelKyZq-pEF1Y